src/i18n/parseFloat.js

```diff
@@ -0,0 +1,12 @@
+// NOTE: This module implements a localized parseFloat function
+// that can be used to parse user input into correct floating point numbers.
+import numeral from "numeral";
+import parseDecimalNumber from "parse-decimal-number";
+import i18next from "./index";
+
+const { language } = i18next;
+// NOTE: I'm not sure if it's fine to split by the IETF language tag by -
+// to get the locale... numeral, however, only takes that, not the minus.
+const options = numeral.localeData(language.split("-")[0]).delimiters;
```

Comment:
This one seems to work only with 'en-*' locale.
All the rest throw an error of unknown locale.

One way to fix it would be to use different option provider, like this for example:
```
import cldr = require('cldr');
import i18next from "./index";

const { language } = i18next;
const [ main, domain ] = language.split("-");
const locale = [main, domain.toUppercase()].join('_');
const options = cldr.extractNumberSymbols(locale);
 
const parse = parseDecimalNumber.withOptions(options);
```

*Got that 👆 piece of code from here :https://www.npmjs.com/package/parse-decimal-number*

Reply:
OK, lemme test with another locale...

Reply:
Resolved in #118